x/pairing/keeper/scores/score_req.go

```diff
@@ -0,0 +1,159 @@
+package scores
+
+import (
+	"fmt"
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/lavanet/lava/utils"
+	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
+	scorestypes "github.com/lavanet/lava/x/pairing/types/scores"
+	planstypes "github.com/lavanet/lava/x/plans/types"
+	tendermintcrypto "github.com/tendermint/tendermint/crypto"
+)
+
+var uniformStrategy scorestypes.ScoreStrategy
+
+// TODO: currently we'll use weight=1 for all reqs. In the future, we'll get it from policy
+func init() {
+	uniformStrategy = make(scorestypes.ScoreStrategy)
+	uniformStrategy[scorestypes.STAKE_REQ_NAME] = 1
```

Comment:
lets use the type instead of defining a string

Reply:
done. see https://github.com/lavanet/lava/pull/594/commits/b0c34d4222b4a24e26081ea469300481dafa0f5d